pkg/documents/documents.go

```diff
@@ -246,6 +264,24 @@ func NewDocumentRepo(logger log.Logger, db *sql.DB) DocumentRepository {
 		logger: logger,
 	}
 }
+
+func (r *sqlDocumentRepository) findActiveDocument(customerID string, documentID string, namespace string) error {
+	query := `select documents.document_id from documents
+inner join customers on customers.namespace = ? where documents.customer_id = ? and documents.document_id = ? and documents.deleted_at is null
+limit 1;`
+	stmt, err := r.db.Prepare(query)
+	if err != nil {
+		return fmt.Errorf("getCustomerDocuments: prepare %v", err)
+	}
+	defer stmt.Close()
+
+	var docID string
+	if err := stmt.QueryRow(namespace, customerID, documentID).Scan(&docID); err != nil {
+		return err
+	}
+	return nil
```

Comment:
Shouldn't `findActiveDocument` return `(*client.Document, error)`? Here the `docID` never gets used.

Reply:
I really just wanted to ensure the document existed and wasn't deleted. Maybe  we should give the method a better name?

Reply:
Ah I see. `exists(...) (bool, err)` could work; and the call would look like:
```go
docExists, err := documentRepo.exists(..)
```

Reply:
that's much better. I'll do that